src/main/java/com/shapesecurity/bandolier/es2017/Bundler.java

```diff
@@ -44,9 +48,8 @@
 	 * @return a bundled script
 	 * @throws ModuleLoaderException
 	 */
-	public static @NotNull
-	Script bundle(@NotNull Path filePath) throws ModuleLoaderException {
-		return bundle(filePath, new FileSystemResolver(), new FileLoader(), new StandardModuleBundler());
+	public static @NotNull Script bundle(@NotNull BundlerOptions options, @NotNull Path filePath) throws ModuleLoaderException {
```

Comment:
Can we keep one that doesn't require `BundlerOptions` and just uses some reasonable default?